lms/templates/verify_student/test/fake_softwaresecure_response.html

```diff
@@ -0,0 +1,76 @@
+<html>
+<head><title>Fake Software Secure Form</title>
+</head>
+<body>
+<p>Fake Software Secure page</p>
+<p>It will pick the most recent software secure attempt for a logged-in user.</p>
+
+<br>
+<div>
+<input type="button" value="PASS" id="btn_pass">
+<input type="button" value="FAIL" id="btn_denied" style="margin-left: 10px">
+<input type="button" value="SYSTEM FAIL" id="btn_error" style="margin-left: 10px">
+<input type="button" value="UNKNOWN ERROR" id="btn_unknown_error" style="margin-left: 10px">
+</div>
+
+<div id="errors-info" style="padding-top: 20px;color: red"></div>
+<div id="success-info" style="padding-top: 20px;color: green"></div>
+
+</body>
+</html>
+<title>Fake Software Secure Response</title>
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>
```

Comment:
Please use the version of JQuery served by the platform.  You should be able to do:
```
<script type="text/javascript" src="${static.url("js/vendor/jquery.min.js")}"></script>
```